pegasus/sites.v3/code.org/views/video_player.js.haml

```diff
@@ -1,13 +1,12 @@
-%script{src: "/js/videojs-ie8.min.js"}
-%script{src: "/js/video.min.js"}
-
 %link{href: "/css/video-js.css", rel: "stylesheet"}
```

Comment:
Yes, this can be done.  I can do it with either jQuery or document.createElement.  Depending on what is decided with the loading of the scripts I will do it the same way.